lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -314,6 +318,14 @@ def configure(conf)
             @resource.type = DATAFLOW_CONSTANTS[:resource_type]
           elsif @subservice_name == ML_CONSTANTS[:service]
             @resource.type = ML_CONSTANTS[:resource_type]
+          elsif @subservice_name = DATAPROC_CONSTANTS[:service]
```

Comment:
There's also a bug here:
```
lib/fluent/plugin/out_google_cloud.rb:321:34: W: Assignment in condition - you probably meant to use ==.
          elsif @subservice_name = DATAPROC_CONSTANTS[:service]
```
but since you're removing this block, the bug is not really relevant, right?